### ISSUE TITLE
dataset issue

### ISSUE THREAD

**Issue by wj7486** [ali-vilab/ACE#13]
I would like to confirm the construction of the training data, such as TryOn. According to the training instructions, six fields are required:

"edit_image": represents the input image for the editing task. If it is not an editing task but a reference generation, this field can be left empty.
"edit_mask": represents the input image mask for the editing task, used to specify the editing area. If it is not an editing task but rather for reference generation, this field can be left empty.
"ref_image": represents the input image for the reference image generation task;  if it is a pure editing task, this field can be left empty.
"target_image": represents the generated target image and cannot be empty.
"prompt": represents the prompt for the generation task.
"data_type": represents the type of data, which can be 'portrait', 'subject', or 'local'. This field is not used in training phase.

The six fields corresponding to the dataset I have prepared are as follows:
"edit_image": 
![Image](https://github.com/user-attachments/assets/843080ff-d588-4535-9f82-bd373d4e4b0e)
"edit_mask": 
![Image](https://github.com/user-attachments/assets/0a8762ab-27f3-41c4-ba16-ea460f7730b8)
"ref_image":
![Image](https://github.com/user-attachments/assets/ac681409-7650-47a5-91f9-d5545d8f5c2a)
"target_image":
![Image](https://github.com/user-attachments/assets/8aec3257-ce95-4069-a9d3-ace3430a3d09)
"prompt": 
Change the cloth in {image} to the one in {image1}
"data_type": 
subject

May I ask if the construction of this dataset is correct

**Comment by wj7486** [comment]
When training for 200 steps, the loss increases and the training fails to obtain the image:
edit_image0：
![Image](https://github.com/user-attachments/assets/ef11dd16-e455-49ab-a880-78b2250c395b)
modify_image0：
![Image](https://github.com/user-attachments/assets/bfd8af5b-b1e2-4b73-990a-3d3a2761185f)
edit_mask0
![Image](https://github.com/user-attachments/assets/fa57b18b-a3f1-43f2-ae20-2d09d0f1c3e3)
target_image
![Image](https://github.com/user-attachments/assets/7c701a40-0452-4860-84c5-f330985b3d52)
result:
![Image](https://github.com/user-attachments/assets/4dde8cdf-403f-4326-9ff3-0df579cb3290)

What problem is causing this? We look forward to your reply
